src/pages/DistributionsPage/DistributionForm.tsx

```diff
@@ -292,259 +300,374 @@ export function DistributionForm({
-              <Box css={{ display: 'flex', justifyContent: 'center' }}>
-                {fixedPaymentTokenSel.length ? (
-                  <Button
-                    color="primary"
-                    outlined
-                    size="medium"
-                    disabled={submitting || !sufficientFixedPaymentTokens}
-                    fullWidth
-                  >
-                    {sufficientFixedPaymentTokens
-                      ? submitting
-                        ? 'Submitting...'
-                        : 'Submit Distribution'
-                      : 'Insufficient Tokens'}
-                  </Button>
-                ) : (
-                  <Button
-                    type="button"
-                    color="primary"
-                    outlined
-                    size="medium"
-                    fullWidth
-                    onClick={async () => {
-                      // use the authed api to download the CSV
-                      if (epoch.number) {
-                        const csv = await downloadCSV(epoch.number);
-                        if (csv?.file) {
-                          const a = document.createElement('a');
-                          a.download = `${circle?.organization.name}-${circle?.name}-epoch-${epoch.number}.csv`;
-                          a.href = csv.file;
-                          a.click();
-                          a.href = '';
-                        }
-                      }
-                      return false;
-                    }}
-                  >
-                    Export CSV
-                  </Button>
-                )}
-              </Box>
-            )}
+                </Box>
+              </TwoColumnLayout>
 
-            {submitting && <LoadingModal visible />}
-          </Box>
-        )}
+              {submitting && <LoadingModal visible />}
+            </>
+          )}
+        </Panel>
+        {(fixedDist || circleDist) && <Summary distribution={fixedDist} />}
+        <Flex css={{ justifyContent: 'center', mt: '$xl', mb: '$xl' }}>
+          {!fixedDist ? (
+            fixedPaymentTokenSel.length ? (
+              <Button
+                color="primary"
+                outlined
+                size="large"
+                disabled={submitting || !sufficientFixedPaymentTokens}
+                fullWidth
+              >
+                {sufficientFixedPaymentTokens
+                  ? submitting
+                    ? `Submitting...`
+                    : `Submit ${fixedPaymentTokenSel[0].symbol} Vault Distribution`
+                  : `Insufficient Tokens`}
+              </Button>
+            ) : (
+              <Button
+                type="button"
+                color="primary"
+                outlined
+                size="large"
+                fullWidth
+                onClick={async () => {
+                  // use the authed api to download the CSV
+                  if (epoch.number) {
```

Comment:
if this doesn't work unless an epoch number is present, we probably just shouldn't show the button at all if it's missing. i see it was like this already before your changes, i should have caught it earlier

Reply:
i've updated it to not show but had to leave the `if (epoch.number)` inside the onclick not sure why typescript is showing an issue